libs/ui-watt/src/lib/components/form-field/form-field.component.scss

```diff
@@ -31,6 +31,13 @@ watt-form-field {
     .mat-form-field-flex {
       height: 56px;
     }
+
+    [wattPrefix],
+    [wattSuffix] {
+      .watt-icon-button {
+        height: 52px;
```

Comment:
issue: this is applying height to icons inside a `watt-icon-button` component. Often the prefix icon is a `watt-icon` component (e.g. inside `dh-metering-point-search-form.component.html`). This style won't be applied in that case. Will there be a case where the prefix icon is a `watt-icon-button`?